EarlGrey/Core/GREYKeyboard.m

```diff
@@ -36,9 +37,9 @@
 static GREYTapAction *gTapKeyAction;
 
 /**
- *  Flag set to @c YES when the keyboard is shown, @c NO when keyboard is hidden.
+ *  Flag set to @c true when the keyboard is shown, @c false when keyboard is hidden.
  */
-static BOOL gIsKeyboardShown = NO;
+static atomic_bool gIsKeyboardShown = false;
```

Comment:
this is cool, good to know, just be mindful that `stdatomic.h` is C11 standard (just looked, xcode7 should have already supported it), but I like the fact we are chasing the latest standard.